service/experiment-requests.yaml

```diff
@@ -19,6 +19,11 @@
 # are still testing this feature. You should request an experiment by contacting
 # us as you normally do.
 
+- experiment: 2020-07-31
```

Comment:
When did you merge the last experiment request? It's probably better to combine them.

Reply:
Last one was requested yesterday, and it's already running.

Reply:
Nevermind then!